lib/ansible/plugins/filter/to_json.yml

```
@@ -0,0 +1,69 @@
+DOCUMENTATION:
+  name: to_json
+  author: core team
+  version_added: 'historical'
+  short_description: Convert variable to JSON string
+  description:
+    - Converts an Ansible variable into a JSON string representation.
+    - This filter functions as a wrapper to the Python C(json.dumps()) function.
+    - Ansible internally auto-converts JSON strings into variable structures so this plugin is used to force it into a JSON string.
+  options:
+    _input:
+      description: A variable or expression that returns a data structure
```

```suggestion
      description: A variable or expression that returns a data structure.
```
Same for all other descriptions (except short_description). I won't create more suggestions for these.